packages/@react-spectrum/table/src/Resizer.tsx

```diff
@@ -32,29 +39,52 @@ function Resizer<T>(props: ResizerProps<T>, ref: RefObject<HTMLInputElement>) {
   useVirtualizerContext();
   let stringFormatter = useLocalizedStringFormatter(intlMessages);
   let {direction} = useLocale();
+  let [directionClass, setDirectionClass] = React.useState(null);
+
+  let [isPointerDown, setIsPointerDown] = useState(false);
+  useEffect(() => {
+    let setDown = () => setIsPointerDown(true);
+    let setUp = () => setIsPointerDown(false);
+    document.addEventListener('pointerdown', setDown, {capture: true});
```

Comment:
Oh maybe what's happening on touch is that this is firing when the user puts their finger down again so the overlay comes back instead of exiting resize mode?

Reply:
ahhhh, thanks for the info, i was just looking into this scratching my head